### PR TITLE
Add EIP: Contract size limit increase for EOF

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: EIPS/eof-contract-size.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+eip: nnnn
```
Comment (axic) [inline EIPS/eof-contract-size.md:2]: @abcoathup please assign a number :)

File: EIPS/eof-contract-size.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+title: Contract size limit increase for EOF
+description: Raise the limit for only EOF contracts to 64 KiB
+author: Alex Beregszaszi (@axic), Danno Ferrin (@shemnon)
+discussions-to:
```
Comment (axic) [inline EIPS/eof-contract-size.md:6]: Will add this after number assignment.

File: EIPS/eof-contract-size.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+requires: 170, 3540, 3860
+---
+
+## Abstract
+
+Revise the contract size limit for EOF contracts to be 64 KiB instead of the existing 24 KiB limit. Legacy contracts are unaffected.
+
+## Motivation
+
+The contract size limit is was introduced as a measure against DoS attacks. `JUMPDEST`-analysis os required for legacy contracts, and many of the algorithms performing it are not linear and/or have unknown unknowns. This is one of the reasons for the hesitance of a limit increase.
```
Comment (abcoathup) [inline EIPS/eof-contract-size.md:20]:
```suggestion
The contract size limit was introduced as a measure against DoS attacks. `JUMPDEST`-analysis is required for legacy contracts, and many of the algorithms performing it are not linear and/or have unknown unknowns. This is one of the reasons for the hesitance of a limit increase.
```

File: EIPS/eof-contract-size.md (hunk 1 above)
Comment (abcoathup) [inline EIPS/eof-contract-size.md:2]:
```suggestion
eip: 7830
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

**Comment by axic** [comment]
@g11tech @gcolvin @lightclient @SamWilsn this is ready for review now

**Review by g11tech — APPROVED** [review]

**Pull request merged**